snippets/ztp-example-siteconfig.adoc

```diff
@@ -77,6 +80,7 @@ spec:
 <4> Cluster labels must correspond to the `bindingRules` field in the `PolicyGenTemplate` CRs that you define. For example, `policygentemplates/common-ranGen.yaml` applies to all clusters with `common: true` set, `policygentemplates/group-du-sno-ranGen.yaml` applies to all clusters with `group-du-sno: ""` set.
 <5> Optional. The CR specifed under `KlusterletAddonConfig` is used to override the default `KlusterletAddonConfig` that is created for the cluster.
 <6> For single-node deployments, define a single host. For three-node deployments, define three hosts. For standard deployments, define three hosts with `role: master` and two or more hosts defined with `role: worker`.
+<7> Specify custom roles to your nodes in your managed clusters. These are additional roles which are not used by any {product-title} components, only by the user. When you add a custom role, it can be associated with a custom machine config pool that references a specific configuration for that role. When you set a custom label on a node, the node reboots and the machine config pool configuration is applied on that node.
```

Comment:
The purpose of this feature is to avoid unnecessary reboots. Can you change "When you set a custom label on a node, the node reboots and the machine config pool configuration is applied on that node." to something like "Integrating custom labels (or roles) during installation enhances the efficiency of the deployment process by eliminating the need for unnecessary reboots after installation."

Reply:
Corrected.